package.json

```diff
@@ -58,6 +58,7 @@
     "@material-ui/core": "^1.5.1",
     "@material-ui/icons": "^3.0.1",
     "JSONStream": "^1.3.5",
+    "appdmg": "^0.4.5",
```

Comment:
```suggestion
```

This shouldn't be committed since it's a Mac only dep. On the todo list: figure out how to stop this getting added to package.json on `npm install`